pom.xml

```diff
@@ -36,12 +36,13 @@
       <!-- Only required if names of spec classes don't match default Surefire patterns (`*Test` etc.) -->
       <plugin>
         <artifactId>maven-surefire-plugin</artifactId>
-        <version>3.0.0-M4</version>
+        <version>3.0.0-M5</version>
         <configuration>
+          <useModulePath>false</useModulePath> <!-- https://issues.apache.org/jira/browse/SUREFIRE-1809 -->
           <useFile>false</useFile>
           <includes>
-            <include>**/*Test.java</include>
-            <include>**/*Spec.java</include>
+            <include>**/*Test</include>
+            <include>**/*Spec</include>
```

Comment:
I remember that in the past ([2005](https://solidsoft.wordpress.com/2015/03/09/spock-1-0-with-groovy-2-4-configuration-comparison-in-maven-and-gradle/)) it was required to have `.java`, but I assume you checked it with 3.0.0 and it works fine.

Reply:
Yeah both work fine, this seems less confusing